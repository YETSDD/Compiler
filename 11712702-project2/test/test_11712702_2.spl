int add(){
    return 1;

}
char add(){
    return 'a';
}
int test_11712702_2()
{
  int i=0,j=0;
  i = j + 1;
  return add(i);
}
