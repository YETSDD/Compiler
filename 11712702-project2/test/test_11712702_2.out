Error type 4 at Line 5: function add is redefined
Error type 9 at Line 12: function add recieves wrong number of args 
Error type 8 at Line 12: incompatiable return type
